randovania/games/dread/json_data/Dairon.txt

```diff
@@ -90,13 +90,11 @@ Extra - asset_id: collision_camera_001
   > Top Floor
       Trivial
   > Event - Speed Booster blocks Destroyed
-      Any of the following:
-          All of the following:
-              # Regular speed charge possible with 1.0 second charge time
-              Speed Booster and Speed Booster Upgrade ≥ 2 and Before Dairon - Teleport To Artaria Speed Blocks Destroyed
-          All of the following:
-              # Short boost speed charge possible with 1.25 second charge time
-              Speed Booster Upgrade and Before Dairon - Teleport To Artaria Speed Blocks Destroyed and Short Boost (Intermediate) and Can Short Boost
+      All of the following:
+          Speed Booster and Before Dairon - Teleport To Artaria Speed Blocks Destroyed
+          Any of the following:
+              Speed Booster Upgrade ≥ 2
+              Speed Booster Upgrade and Short Boost (Intermediate) and Can Short Boost
```

Comment:
Would it make sense to add the comments here again or were/are they to generic?

Reply:
The "Before" event can also be removed here. Logic would understand if it is already destroyed, and if it's already destroyed there is no reason to charge the speed either.

Reply:
I'd have to find one of the other instances of where I put a comment to copy it :P

I don't honestly recall why I included the "before" criteria; I probably thought "can't hurt" and added it, but I can remove it.